DirectProgramming/DPC++FPGA/Tutorials/Features/lsu_control/lsu_control.sln

```diff
@@ -17,10 +15,6 @@ Global
 		{D6A634E7-9F2B-46C2-A21C-2402F631A55A}.Debug|x64.Build.0 = Debug|x64
 		{D6A634E7-9F2B-46C2-A21C-2402F631A55A}.Release|x64.ActiveCfg = Release|x64
 		{D6A634E7-9F2B-46C2-A21C-2402F631A55A}.Release|x64.Build.0 = Release|x64
```

Comment:
Should these lines be deleted too?

Reply:
Nope, in this case, those targets have the id: D6A634E7-9F2B-46C2-A21C-2402F631A55A which corresponds to the lsu_control.vcxproj file that is kept.

Reply:
I see. Thanks.